### DISCUSSION TITLE
How can i show resource in dashboard?

### DISCUSSION THREAD

**Discussion by beshoo** [laravel/nova-issues#5627] [Q&A] [answered]
- Laravel Version: 10
- Nova Version: [v4.25.1 (Silver Surfer)]
- PHP Version: 8.1
- Database Driver & Version: Mysql 8
- Operating System and Version: Linux
- Browser type and version: Chrome

### Description:
I want to make something like this:
![image](https://github.com/laravel/nova-issues/assets/4635115/6c9099a3-8a1c-4b3b-9932-2bf4448f68b6)

It seems the Dashboard only accepts cards.
I don't want to redirect users to a resource. how can we show the resource in the dashboards?

**Comment by crynobone** [comment]
The screenshot is from a Resource Index page (with cards) and not a Dashboard.

**Reply by beshoo** [reply]
Thanks for your reply. I'm talking about the screenshot and how the dashboard should look like. Can we make resources the main section?

**Comment by crynobone** [comment, marked as answer]
You can set a resource as the main initial path after login, check the documentation: https://nova.laravel.com/docs/4.0/installation.html#customizing-nova-s-initial-path
